packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -205,20 +210,32 @@ function registerCommands(
     'sfdx.force.apex.test.run',
     forceApexTestRun
   );
-  const forceApexExecuteDocumentCmd = vscode.commands.registerCommand(
+  const forceAnonApexExecuteDocumentCmd = vscode.commands.registerCommand(
     'sfdx.force.apex.execute.document',
-    () => forceApexExecute(false)
+    forceAnonApexExecute
+  );
+  const forceApexDebugDocumentCmd = vscode.commands.registerCommand(
+    'sfdx.force.apex.debug.document',
+    forceAnonApexDebug
```

Comment:
Instead of passing true/false to forceAnonApexExecute(), we now have forceAnonApexExecute() and forceAnonApexDebug()